### PR TITLE
Database migrations

### PR DESCRIPTION
**Pull request by dianpan**
@arf563 
add db migrations

### CODE REVIEW

File: db/migrate/20151025194737_create_stocks.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateStocks < ActiveRecord::Migration
+  def change
+    create_table :stocks do |t|
+      t.string :symbol, null:false
```
Comment (arf563) [inline db/migrate/20151025194737_create_stocks.rb]: I think this should have index: true

File: db/migrate/20151025194751_create_quotes.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class CreateQuotes < ActiveRecord::Migration
+  def change
+    create_table :quotes do |t|
+      t.references :stock, index: true, foreign_key: true, null: false
+      t.date :date, index: true, null: false
+      t.float :open
+      t.float :close
+      t.float :high
+      t.float :low
+      t.integer :volume
```
Comment (arf563) [inline db/migrate/20151025194751_create_quotes.rb:10]: Is this 32 or 64 bit integer?

Comment (dianpan) [inline db/migrate/20151025194751_create_quotes.rb:10]: 32 bit

File: db/migrate/20151025194751_create_quotes.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class CreateQuotes < ActiveRecord::Migration
+  def change
+    create_table :quotes do |t|
+      t.references :stock, index: true, foreign_key: true, null: false
```
Comment (arf563) [inline db/migrate/20151025194751_create_quotes.rb:4]: I think this table should have a composite key of stock,date.

https://github.com/composite-primary-keys/composite_primary_keys

File: db/migrate/20151025194751_create_quotes.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      t.date :date, index: true, null: false
+      t.float :open
```
Comment (arf563) [inline db/migrate/20151025194751_create_quotes.rb]: If these change to integers, let's change all the names to include cents:

open_cents
close_cents
high_cents

etc.

We can also add a string "currency" field with default value "USD"

File: db/migrate/20151025194737_create_stocks.rb (hunk 1 above)
Comment (dianpan) [inline db/migrate/20151025194737_create_stocks.rb]: will add

**Comment by arf563** [comment]
LGTM

**Pull request merged**